react-native/react/actions/tracker.js

```diff
@@ -18,6 +18,30 @@ type TrackerActionCreator = (dispatch: Dispatch, getState: () => {tracker: RootT
 
 // TODO make actions for all the call back stuff.
 
+export function registerTrackerChangeListener (): (dispatch: Dispatch) => void {
+  return dispatch => {
+    const param = {
+      channels: {
+        tracking: true
+      }
+    }
+    engine.listenOnConnect(() => {
+      engine.rpc('notifyCtl.setNotifications', param, {}, (error, response) => {
+        if (error != null) {
+          console.error('error in toggling notifications: ', error)
+        }
+      })
+    })
+
+    engine.listenGeneralIncomingRpc('keybase.1.NotifyTracking.trackingChanged', function (args) {
```

Comment:
The only thing the code path does is fire an action whose reducer sets `trackers: [username]: closed: true`, so that'd happen twice.. I think that'd be safe.